dbt_project.yml

```diff
@@ -7,8 +7,8 @@ profile: default
 vars:
 
   ## Set the database and schema where raw CCLF data is stored
-  input_database: default
-  input_schema: default
```

Comment:
Was this on purpose or just left over from testing?

Reply:
@sarah-tuva this comment is on the 'before' diff.   The dbt_project.yml currently points to CI which I think is okay.  I don't want to rerun CI and the schema still says CCLF